libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -102,6 +108,16 @@ class CppInterface : public testing::Test {
     return module.GetErrorMessage();
   }
 
+  // Compiles a shader, expects compilation success, and returns the output
+  // bytes.
+  std::string CompilationOutput(const std::string& shader,
```

Comment:
Ah. You use this for both binary and disassembled compilation.
In that case I'd recommend making two methods, one for each output form.